source/slang/hlsl.meta.slang

```diff
@@ -9880,7 +9880,7 @@ float3 HitTriangleVertexPositions(uint index)
                 OpExtension "SPV_KHR_ray_tracing";
                 OpExtension "SPV_KHR_ray_tracing_position_fetch";
                 %_ptr_Input_v3float = OpTypePointer Input $$float3;
-                %addr = OpAccessChain %_ptr_Input_v3float builtin(HitTriangleVertexPositionsKHR:float3[3]) $index;
+                %addr = OpAccessChain builtin(HitTriangleVertexPositionsKHR:float3[3]) $index;
```

Comment:
This needs to be `%addr : %_ptr_Input_v3float = ...`

Reply:
yea, noticed that, resolved in the latest commit.